lib/ui/home/widgets/animated_percentage_widget.dart

```diff
@@ -56,5 +57,5 @@ class _AnimatedPercentageWidgetState extends State<AnimatedPercentageWidget>
 
   @override
   Widget build(BuildContext context) =>
-      Text('$_number%', style: Theme.of(context).textTheme.headline6);
+      Text(Strings.of(context).percentage(_number), style: Theme.of(context).textTheme.headline6);
```

Comment:
dałbym przecinek na końcu, żeby się sformatowało. Ale to bardziej kwestia dogadania się jakiej konwencji chcemy się trzymać

Reply:
Nie patrzyłem nawet jak to wygląda na UI :)
Po prostu przeniosłem istniejący tekst, looknę i dam znać.